.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,16 @@
+name: Basic pipeline
+on: push
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
```

Comment:
We will also want to run `go vet` as well as `go test`

Reply:
Left out `go test` for now since we don't have any tests.